### ISSUE TITLE
Enable controller 2

### ISSUE THREAD

**Issue by eduardonot** [nbarkhina/N64Wasm#11]
Is it possible to enable a second controller?

**Comment by nbarkhina** [comment]
@eduardonot yes it's on my todo list to implement a second controller feature

**Comment by nbarkhina** [comment]
Multiple controller support has now been implemented
